stable/nats/values.yaml

```diff
@@ -250,36 +250,45 @@ ingress:
   #   key:
   #   certificate:
 
-sidecars:
-## Add sidecars to the pod.
-## e.g.
-# - name: your-image-name
-  # image: your-image
-  # imagePullPolicy: Always
-  # ports:
-  # - name: portname
-  #   containerPort: 1234
-
-## Prometheus Exporter / Metrics
+## Metrics / Prometheus NATS Exporter
 ##
+## ref: https://github.com/nats-io/prometheus-nats-exporter
 metrics:
   enabled: false
   image:
     registry: docker.io
-    repository: appcelerator/prometheus-nats-exporter
-    tag: 0.17.0
+    repository: synadia/prometheus-nats-exporter
```

Comment:
What are the main advantages of using this new exporter?

Reply:
I see the main advantages as:
- using the "official" dockerhub repo maintained by Synadia
- having an image tag that corresponds to the [`0.1.0`](https://github.com/nats-io/prometheus-nats-exporter/releases/tag/v0.1.0) release of `prometheus-nats-exporter`

Reply:
It seems it's the official one: https://github.com/nats-io/nats-operator/tree/master/helm/nats-operator

I'm okay with replacing it!